src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -x
```

Comment:
This might generate lots of traces on regular runs, what about conditionally setting this (when needed you can simply inject the env. variable in the Jenkins jobs):
```suggestion
[ -v SHELL_TRACE ] && set -x
```